File: CustomDialogBox/app/src/main/java/com/example/customdialogbox/MainActivity.java
package com.example.customdialogbox;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.AppCompatButton;

import android.app.Dialog;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Dialog dialog= new Dialog(MainActivity.this);
        dialog.setContentView(R.layout.custom_dialog_layout);
        dialog.setCancelable(false);
        AppCompatButton btnok = dialog.findViewById(R.id.btnok);

        btnok.setOnClickListener(view -> {
            Toast.makeText(MainActivity.this, "Closed", Toast.LENGTH_SHORT).show();
            dialog.dismiss();
        });

        dialog.show();
    }
}

File: KotlinCalculator/app/src/main/java/com/example/kotlincalculator/MainActivity.kt
package com.example.kotlincalculator

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.widget.AppCompatButton
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        btnAdd.setOnClickListener {
            if (edtNo1.text.toString() != "" && edtNo2.text.toString() != ""){
                val no1= edtNo1.text.toString().toInt()
                val no2= edtNo2.text.toString().toInt()
                val sum = no1 + no2
                Toast.makeText(this,"The Sum is $sum", Toast.LENGTH_LONG).show()
                txtSum.text = "The Sum is $sum"
        }}
    }
}


